style/modules/grid.css

```diff
@@ -0,0 +1,20 @@
+.gridContainer{ /* a container with grid */
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-gap: 10px;
+    background: #F5F5F5;
+    padding: 10px;
+}
+
+.gridContainer * {  /* applies to all the items in the grid */
```

Comment:
I haven't tried it but I think `.gridContainer > *` will work and if so it's better because it only targets the direct children